doc/user_manual/chSpecialElements.tex

```diff
@@ -1652,23 +1652,290 @@ \section{Tracking Code for Collimation Studies} \label{sec:collimat}
+This feature will be deprecated in the future, and the functionality taken over by an extended \texttt{DIST} module.
+The different options are listed below.
+
+\begin{enumerate}
+    \setcounter{enumi}{-1}
+    \item The internal collimation distribution generator is disabled and the general SixTrack distribution is used instead (default).
+    \item Distribution in the plane for which the parameters are specified ONLY: flat distribution in the selected plane between $A_x \pm \delta A_x$ (horizontal) or $A_y \pm \delta A_y$ (vertical). The amplitude in the other plane is zero\index{beam distribution}. The parameters must be provided with the \texttt{DIST\_PARAM} keyword. Values 1 to 4 are used.
+    \item Distribution in the plane for which the parameters are specified + a Gaussian distribution cut at $3\sigma$ in the other plane. The parameters must be provided with the \texttt{DIST\_PARAM} keyword. Values 1 to 4 are used.
+    \item Distribution in the plane for which the parameters are specified + a Gaussian distribution cut at $3\sigma$ in the other plane. The parameters must be provided with the \texttt{DIST\_PARAM} keyword. Values 1 to 4 are used. In addition and energy spread is given value 5, and a longitudinal component given by value 6.
+    \item Reads an external file that contains the beam distribution to be tracked. The file is specified by \texttt{DIST\_FILE}.
+    \item Radial transverse distribution of radius $A_r$. This corresponds to a flat distribution both in the horizontal and vertical planes between $A_x \pm \delta A_x$ and $A_y \pm \delta A_y$, with $A_x = A_y = A_r/\ddot{O}2$.
+    \item Reads a normalised distribution from an external file. The file is specified by \texttt{DIST\_FILE}.
+\end{enumerate}
+
+\paragraph{Distribution Generator Parameters} \texttt{DIST\_PARAM $A_x$ $\delta A_x$ $A_y$ $\delta A_y$ [enerror bunchlen]}\\
+
+The parameters for the distribution generator \texttt{DIST\_TYPE}.
+The required values are determined by the type selected.
+All values default to 0.
+
+Nominal values are $3.06 \times 10^{-4}$ at 450 GeV and $1.129 \times 10^{-4}$ at 7 TeV for \texttt{enerror}, and 11.24 cm at 450 GeV and 7.55 cm at 7TeV for \texttt{bunchlen}.
+
+\paragraph{Distribution Generator Input File} \texttt{DIST\_FILE filename}\\
+
+Name of the distribution file to be read if for distribution type 4 or 6. 
+
+\paragraph{Special Radial Distribution} \texttt{DO\_RADIAL true|false size smear}\\
+
+Alternative radial beam distribution where \texttt{size} is the size of the beam in numbers of radial sigmas, and \texttt{smear} is the smear of the beam in radial sigma.
+Setting this to true will oveeride \texttt{DIST\_TYPE}.
+
+\paragraph{Do Change Collimator Gaps} \texttt{DO\_NSIG true|false}\\
+
+If true, the collimator gaps specified in the imput block by keyword \texttt{NSIG\_FAM} are used instead of those in the collimator database file.
```

Comment:
I will when the new DB format is implemented. It would be misleading to say anything here as there is no other priorities than what is already stated in the text.